x-pack/heartbeat/monitors/browser/source/local.go

```diff
@@ -56,7 +56,7 @@ func (l *LocalSource) Fetch() (err error) {
 	if l.workingPath != "" {
 		return nil
 	}
-	l.workingPath, err = ioutil.TempDir("/tmp", "elastic-synthetics-")
+	l.workingPath, err = ioutil.TempDir(os.TempDir(), "elastic-synthetics-")
```

Comment:
This makes running tests natively on windows possible, and is more correct anyway